### ISSUE TITLE
Blaze Budget Screen Redesign

### ISSUE THREAD

**Issue by selanthiraiyan** [woocommerce/woocommerce-ios#12538]
Clone of Android issue https://github.com/woocommerce/woocommerce-android/issues/11322

**To be tackled as part of Blaze i4 Project**

During the User Flow Review session: pe5sF9-2Bb-p2 there was a lot of discussion revolving the current budget screen: 

<img width="200"  src="https://github.com/woocommerce/woocommerce-android/assets/2663464/56362a90-c046-4032-a4f2-24c6ba3b822d">

This design presents a number of issues. Following are the suggestions shared during the session: 

- It seems daily spending should be highlighted more. It was discussed that this could potentially be the most relevant number in the screen. Specially if, in the future, we want to support “Evergreen” (aka campaigns with no end date) campaigns.
- Campaign reach/impressions seem to also be one of the most relevant figures on the screen and are not highlighted enough.
- The current design doesn't support `Evergreen` campaigns (campaigns with no end date)

When we tackle i4 we will work alongside Joe Keenan to restructure the screen data.

**Comment by dangermattic** [comment]
🚫 Please add a type label (e.g. **type: enhancement**) and a feature label (e.g. **feature: stats**) to this issue.

**Comment by JorgeMucientes** [comment]
Hey @itsmeichigo I noticed that this task is marked as done. However some of the points described in the issue where not tackled as we haven made the `daily/weekly spending figures more prominent` or the highlighted the impressions numbers. Afaik on iOS the introduced changes where only to support Evergreen campaigns right? 
I wonder if we should ping Joe to revisit the other 2 points we got back from the user flow review.
